### DISCUSSION TITLE
post/2024/create-an-ai-assistant-with-your-own-data/

### DISCUSSION THREAD

**Discussion by giscus** [Jandev/janvnl-comments#1] [Announcements]
# post/2024/create-an-ai-assistant-with-your-own-data/

The current large language models, like GPT-4, GPT-4 Turbo and GPT-4o are great when you need some output generated based on data you feed in the prompt. Even the small language models, like Phi-3, are doing a great job at this. However, these models often don’t know a lot about the data within your company. Because of this, they can’t do a good job at answering questions that required data from your organization.

https://jan-v.nl/post/2024/create-an-ai-assistant-with-your-own-data/

**Comment by Jandev** [comment]
This is great stuff! Of course, it's all beta material, but really sets a path for the future and how to easily integrate Azure Open AI Assistants to applications.
